src/turno.py

```diff
@@ -20,22 +20,41 @@ def __init__(self, id, enum_cola):
             enum_cola : Enum
                 Enum que indica a que cola pertenece el turno.
         """
-        self.id_cola = id
-        self._check_valid_enum(enum_cola)
-        self.tipo_cola = enum_cola
+        self.id_turno = id
+        enum.TiposTurnos.check_valid_enum(enum_cola)
+        self.enum_turno = enum_cola
+        self.tiempo_turno = 0
 
-    def _check_valid_enum(self, enum_cola):
+    @property
+    def id(self):
         """
-            Validar el tipo de cola que se le esta pasando al enum.
+            Getter del id del turno.
+        """
+        return self.id_turno
 
-            Parameters
-            ----------
-            enum_cola : Enum
-                Enum que indica a que cola pertenece el turno.
-            Raises
-            ------
-            ValueError
-                Si el enum pasado por no es uno válido.
+    @id.setter
```

Comment:
¿En serio que puede cambiar el ID? ¿Qué pasa entonces con todos los que se refieran a este ID?
De veras que el tema de getters y setters tenéis que mirarlo. Pista: en general no hacen falta ninguno de los dos, porque son detalles de implementación que no tienen por qué salir de la clase.